tuf/repository_lib.py

```diff
@@ -1476,7 +1476,7 @@ def generate_snapshot_metadata(metadata_directory, version, expiration_date,
   metadata_directory = _check_directory(metadata_directory)
 
   # Snapshot's 'fileinfodict' shall contain the version number of Root,
-  # Targets, and all delegated roles fo the repository.
+  # Targets, and all delegated roles to the repository.
```

Comment:
Wouldn't "of" make more sense?

Reply:
Yes, a good point.
I am not a native english speaker and sometimes I make mistakes like that one. :D

Reply:
That makes two of use. ;)

Reply:
Oh, no, me three...